### PR TITLE
Add README.md document

### PR DESCRIPTION
**Pull request by ikalnytskyi**

### CODE REVIEW

**Comment by ikalnytskyi** [comment]
@malor Please feel free to suggest paragraphs.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+// Consume a given number of tokens from a bucket. In most cases you want to
+// consume just one token because you're rate limiting events as they come.
+assert_eq!(limiter.consume("A", 1), Ok(()));
+
+// When an event is allowed to occur, Ok() is returned.
+assert_eq!(limiter.consume("A", 1), Ok(()));
+
+// When an event is forbidden to occur, Error:RetryAfter() is returned. The
+// latter contains amount of seconds to wait before the forbidden event is
```
Comment (malor) [inline README.md]: Instead of `amount of seconds`, perhaps, we should just say `Duration` referring to the struct from the standard library?

**Review by malor — APPROVED** [review]
Thank you! :pray:

**Comment by malor** [comment]
Do we need to add LICENSE as well?

**Pull request merged**